src/bin/alert_worker.rs

```diff
@@ -1,10 +1,19 @@
 use boom::{alert, conf, types::ztf_alert_schema, worker_util};
+use clap::Parser;
 use redis::AsyncCommands;
-use std::env;
 use std::sync::{Arc, Mutex};
 use tracing::{error, info, warn, Level};
 use tracing_subscriber::FmtSubscriber;
 
+#[derive(Parser)]
+struct Cli {
+    #[arg(required = true, help = "Name of stream to ingest")]
+    stream: Option<String>,
```

Comment:
making we should make the stream an enum? That way we can error out if the value provided isn't correct

Reply:
What would the values of that enum be?

Reply:
the instruments that we support? So for now just `ZTF`? Later will be ZTF + LSST + ...